bin/lib/ce_install.py

```diff
@@ -281,6 +282,10 @@ def main():
         if num_failed:
             sys.exit(1)
         sys.exit(0)
+    elif args.command == 'reformat':
```

Comment:
We could consider more general formatting: `pre-commit` (which we already use) can (re-)format yaml and enforce style, though this opens the door to our own custom stuff. `pre-commit` means it'll automatically happen, so - we could mix and match if we thought it worthwhile.